### PR TITLE
Add functional tests for add_targets + small improvements

### LINKED ISSUE #47: [Functional tests] Add targets
Use the Targets API to add new targets.
1. Using a valid Token with scope `write:targets` (maybe a fixture for it),
2. Add at least 10 new targets
3. Check the task status
4. Measure the time from sending the request, and the targets are added

Nice to have:
- Use the `python-tuf`, `ngclient`, to see when the targets are added in above item 4.

Requires to close:
- [ ] https://github.com/vmware/repository-service-tuf/pull/116
- [ ] Tests implemented

### PR DESCRIPTION
**Pull request by MVrachev**
The main changes in this pr are:
- adding functional tests for add_targets.feature file
- changes to the add_targets.feature file
- adding get_target_info() pytest feature inside conftest for future testing
- some small fixes 

Closes #47 

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

**Review by rdimitrov — CHANGES_REQUESTED** [review]
The FT need to be updated though 👍

**Comment by MVrachev** [comment]
I have updated the `ngclient` fixture to use the content of the `metadata` directory generated by the CLI test https://github.com/vmware/repository-service-tuf-cli/blob/068c446e67cbed2ecf4250defb7bee6954994ed9/tests/unit/cli/admin/test_ceremony.py#L295
This was possible because of the lastest pr https://github.com/vmware/repository-service-tuf-cli/pull/148.

File: tests/functional/targets/test_add_targets.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+@scenario(
+    "../../features/targets/add_targets.feature",
+    "Adding a target using RSTUF API",
+)
+def test_adding_a_target_using_rstuf_api():
+    """Adding a target using RSTUF API"""
+
+
+@given(
+    "the API requester has a token with a scope write for targets "
```
[the remark this answers is not in the document]
Comment (MVrachev) [reply tests/functional/targets/test_add_targets.py:20]: Yes, but this is for an older commit.
Given that we will squash all of them I don't think we need to fix that.

**Review by kairoaraujo — APPROVED** [review]
LGTM

**Review by rdimitrov — APPROVED** [review]

**Pull request merged**